include/regs.h

```diff
@@ -47,6 +47,11 @@
 #define R_RUN_SPEED_LIMIT                 REG(45)
 #define R_UPDATE_RATE                     SREG(30)
 #define R_PAUSE_MENU_MODE                 SREG(94)
+#define R_PLAY_FILL_SCREEN_ON             MREG(64)
+#define R_PLAY_FILL_SCREEN_R              MREG(65)
+#define R_PLAY_FILL_SCREEN_G              MREG(66)
+#define R_PLAY_FILL_SCREEN_B              MREG(67)
+#define R_PLAY_FILL_SCREEN_ALPHA          MREG(68)
```

Comment:
does mm use this feature at all? in oot this is used only by king dodongo. the "real" screenfill that should be used is the environment context variables that can set color and enable it and stuff. 

if it ended up getting more use in this game then this seems fine, but from oots perspective this feature is more of a hack than anything lmao

Reply:
So far, this system is used by:
- Twinmold
- Song of Soaring Effect
- Koume in Woods of Mystery
- Grandma when telling storytime